libspiped/Makefile.BSD

```diff
@@ -0,0 +1,102 @@
+# Library name.
+LIB	=	spiped
+
+# Don't install it.
+NOINST	=	1
+
+# Useful relative directories
+LIBCPERCIVA_DIR	=	../libcperciva
+LIB_DIR		=	../lib
+PROTO_DIR	=	../proto
+
+# spipe protocol
```

Comment:
s/spipe/spiped/

It's the `spiped` protocol even if it's being used by `spipe`.

Reply:
Also, we should probably move `proto` to `lib/proto`.  But let's do that after we finish reorganizing the build code.